test/distributed/org/apache/cassandra/distributed/test/hostreplacement/AssassinateAbruptDownedNodeTest.java

```diff
@@ -36,4 +40,20 @@ void consume(Cluster cluster, IInvokableInstance nodeToRemove)
     {
         stopAbrupt(cluster, nodeToRemove);
     }
+
+    @Override
+    protected void afterNodeStatusIsLeft(Cluster cluster, IInvokableInstance removedNode)
+    {
+        // Check it is possible to alter keyspaces (see CASSANDRA-16422)
+
+        // First, make sure the node is convicted so the gossiper considers it unreachable
+        InetSocketAddress socketAddress = removedNode.config().broadcastAddress();
+        InetAddressAndPort removedEndpoint = InetAddressAndPort.getByAddressOverrideDefaults(socketAddress.getAddress(),
+                                                                                             socketAddress.getPort());
+        cluster.get(BaseAssassinatedCase.SEED_NUM).runOnInstance(() -> Gossiper.instance.convict(removedEndpoint, 1.0));
+
+        // Second, try and alter the keyspace.  Before the bug was fixed, this would fail as the check includes
+        // unreachable nodes that could have LEFT status.
+        cluster.schemaChangeIgnoringStoppedInstances(String.format("ALTER KEYSPACE %s WITH DURABLE_WRITES = false", KEYSPACE));
```

Comment:
this is the current create logic

```
cluster.schemaChange("CREATE KEYSPACE " + KEYSPACE + " WITH replication = {'class': 'SimpleStrategy', 'replication_factor': " + replicationFactor + "};");
```

we should prob do something like comment, so we do not get bitten by default values changing over time.

Reply:
bad comment, only tables can do that